lib/config.js

```diff
@@ -110,12 +111,54 @@ module.exports = function (cfg) {
   }
 
   // --------------------------------------------------------------------------
-  // Settings: Available Selenium-ish options.
+  // Settings: Extract current desired setting path
   // --------------------------------------------------------------------------
-  if (!cfg.settings.default) {
-    cfg.settings.default = GLOBAL_DEFAULT;
+  var settingsPath = ROWDY_SETTINGS.replace(QUOTES_RE, "").split(".");
+  settingsPath = _.chain(settingsPath)
+    .map(function (v) { return v.trim(); })
+    .filter(_.identity)
+    .value();
+
+  // --------------------------------------------------------------------------
+  // Browser setting extensions (Guacamole)
+  // --------------------------------------------------------------------------
+  if (cfg.options.guacamole) {
+    // Lazy-require.
+    var guacamole = require("guacamole");
+
+    // SauceLabs.
+    if (settingsPath[0] === "sauceLabs") {
+      //
+      // TODO: Fix this path
```

Comment:
Ideas here:

1. Try `cwd() + guacamole-shrinkwrap.json`, then `rowdy/guacamole-shrinkwrap.json`
2. Require a path directly set as `options.guacamoleShrinkwrap` (with a default to `rowdy/guacamole-shrinkwrap.json`) and error without it.

what do you think?